run.go

```diff
@@ -25,6 +25,13 @@ import (
 
 var errUserExit = errors.New("user requested termination")
 
+// maxGrpcMsgSize is used when we configure both server and clients to allow sending and
+// receiving at most 13 MB GRPC messages.
+//
+// This value is based on the default number of forwarding history entries that we'll
+// store in the database, as this is the largest query we currently make.
+const maxGrpcMsgSize = 13 * 1024 * 1024
```

Comment:
Maybe at some extra lee way here? I don't think there is much of a problem by for example setting it to 32 mb?

Reply:
Bumped to 32, in case some super-routers increase their limits above the details. 

Real solution here is to add pagination of course - I'll open up an issue for it so it's not forgotten.